player/command.c

```diff
@@ -930,7 +930,9 @@ static int mp_property_chapter(void *ctx, struct m_property *prop,
                 if (current_chapter_start != MP_NOPTS_VALUE &&
                     get_current_time(mpctx) - current_chapter_start >
                     mpctx->opts->chapter_seek_threshold)
+                {
                     step_all++;
+                }
```

Comment:
Yeah it's been bothering me since I added that code a long time ago. Figured I'd fix it since I'm in the same file and the other changes are minimal / localized enough that it wouldn't confuse anyone. Do you not want it?